src/content/glossary/index.md

```diff
@@ -661,6 +798,10 @@ An attack that consists of an attacker contract calling a victim contract functi
   Re-entrancy
 </DocLink>
 
+### reputation {#reputation}
+
+The property of an identity that other entities believe that identity to be either (1) competent at some specific task, or (2) trustworthy in some context, i.e., not likely to betray others even if short-term profitable.
+
```

Comment:
May push back on this one... as stated this is not really an Ethereum-specific term. Would remove unless we want to rework to be an Ethereum related definition for on-chain reputation, but things are still early on that front so may be best to wait on it for now. 
```suggestion
```